renderdoc/android/android.cpp

```diff
@@ -337,6 +343,49 @@ AndroidVersionCheckResult CheckAndroidServerVersion(const rdcstr &deviceID, ABI
   return AndroidVersionCheckResult::WrongVersion;
 }
 
+Process::ProcessResult InstallAPK(const rdcstr &deviceID, const rdcstr &apk, int apiVersion)
+{
+  Process::ProcessResult adbInstall;
+  if(apiVersion >= 30)
+  {
+    adbInstall = adbExecCommand(deviceID, "install -r -g --force-queryable \"" + apk + "\"");
+  }
+  else
+  {
+    adbInstall = adbExecCommand(deviceID, "install -r -g \"" + apk + "\"");
+  }
+
+  return adbInstall;
+}
+
+AndroidInstallPermissionCheckResult CheckAndroidServerInstallPermissions(const rdcstr &deviceID,
+                                                                         rdcstr packageName,
+                                                                         int apiVersion)
+{
+  // Permissions check only applicable to API > 30
```

Comment:
Very minor nitpick if any changes are made then could this be clarified and changed if needed to be "API >= 30" or "API > 30" : from the code it should be "API >= 30" but the comment might be correct and the code is wrong

Reply:
Good point, I have corrected the comment to "API >= 30"